### ISSUE TITLE
Is it possible to query S###_C##_###E variables from ACS5?

### ISSUE THREAD

**Issue by MaxGhenis** [datamade/census#90]
I'm looking for household income distribution data from the `S1901` table, e.g. `S1901_C01_011E` (households with income >$200k). I can get this from the API directly via https://api.census.gov/data/2018/acs/acs5/subject?get=S1901_C01_011E&for=us:*, but can't find a way to get it through the package.

For example, neither of these work:
```
c.acs5.get('S1901_C01_011E', {'for': 'state:*'})
c.acs5.get('S1901_011E', {'for': 'state:*'})
```
though the published example does:
```
c.acs5.get('B01001_004E', {'for': 'state:*'})
```
Is this possible?

**Comment by Detinator10** [comment]
You can get the subject tables by adding a few lines of code to core.py. For reference, you can see the current pull request to add subject tables (that's what the things that start with s are). However, that has a small typo/issue in that it adds a acs5 to the url when it shouldn't. If you want you can check out my fork, I've implemented the pull request, fixed the typo, and it is currently up-to-date.

**Comment by Detinator10** [comment]
https://github.com/datamade/census/pull/60 is the pull request for reference and kudos to the person who made it, the only issue is the url it defines include acs5 when it shouldn't because acs5 is included automatically.

**Comment by jeancochrane** [comment]
Support for ACS5 subject tables was merged in #92 and released in https://github.com/datamade/census/releases/tag/v0.8.15.
